examples/nidigital/configure-voltage-levels-and-termination-voltage.py

```diff
@@ -0,0 +1,200 @@
+# This example demonstrates how to use the NI-Digital Pattern Driver API to create an
+# instrument session, configure various voltage levels, and then burst a
+# pattern. The vol, voh, vih, vil, iol, ioh and vcom values are configurable parameters and the
+# user can select from High-Z (h), Active Load (a) and Three-Level Drive (t)
+# termination options in termSelect.
+#
+# The gRPC API is built from the C API.  NI-Digital Pattern Driver Pattern documentation is installed with
+# the driver at:
+# C:\Users\Public\Documents\National Instruments\NI-Digital-Pattern-Driver\Documentation\Digital Pattern Help.chm
+#
+# Getting Started:
+#
+# To run this example, install "NI-Digital Pattern Driver" on the server machine.
+# Link :
+# https://www.ni.com/en-in/support/downloads/drivers/download.ni-digital-pattern-driver.html#367315
+#
+# For instructions on how to use protoc to generate gRPC client interfaces, see
+# our "Creating a gRPC Client" wiki page.
+# Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+#
+# Refer to the NI-Digital Pattern Driver gRPC Wiki for the latest C Function Reference:
+# Link: https://github.com/ni/grpc-device/wiki/NI-DIGITAL-PATTERN-DRIVER-C-Function-Reference
+#
+# Running from command line:
+# Server machine's IP address, port number, and resource name can be passed as separate command line arguments.
+#   > python configure-voltage-levels-and-termination-voltage.py <server_address> <port_number>
+# If they are not passed in as command line arguments, then by default the server address will be "localhost:31763"
+import grpc
+import sys
+import os
+import nidigital_pb2 as nidigital_types
+import nidigital_pb2_grpc as grpc_nidigital
+
+server_address = "localhost"
+server_port = "31763"
+session_name = "NI-Digital-Pattern-Driver-Session"
+
+# resource and options for a simulated 6570 client. Change them according to the NI-Digital Pattern Driver model.
+resource = "PXI1Slot2,PXI1Slot3"
+options = "Simulate=1, DriverSetup=Model:6570"
+# Set dir to point to location of shipped pinmap file with gRPC Example
```

Comment:
@harsha-bhushan can you review the comment newly added for location of config files? I think it should finally convey to users that they should provide the absolute location of the folder containing config files **on the server machine**.